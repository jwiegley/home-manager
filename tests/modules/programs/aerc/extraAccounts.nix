{ config, lib, pkgs, ... }:

with lib;

{
  config = {
    nmt.script = let dir = "home-files/.config/aerc";
    in ''
      assertFileContent   ${dir}/accounts.conf ${./extraAccounts.expected}
      assertPathNotExists ${dir}/binds.conf
      assertPathNotExists ${dir}/aerc.conf
      assertPathNotExists ${dir}/stylesets
      assertPathNotExists ${dir}/templates
    '';

    test.stubs.aerc = { };

    programs.aerc = {
      enable = true;

      extraAccounts = {
        Test1 = {
          source = "maildir:///dev/null";
          enable-folders-sort = true;
          folders = [ "INBOX" "SENT" "JUNK" ];
        };
        Test2 = { pgp-key-id = 42; };
      };
    };
  };
}
