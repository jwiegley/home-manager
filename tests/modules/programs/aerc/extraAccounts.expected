# Generated by Home Manager.

[Test1]
enable-folders-sort = true
folders = INBOX,SENT,JUNK
source = maildir:///dev/null

[Test2]
pgp-key-id = 42


